readme.md

```diff
@@ -155,6 +157,28 @@ const counter = new app.locals.Prometheus.Counter({
 counter.inc();
 ```
 
+### `@promster/fastify`
+
+```js
+const app = require('./your-fastify-app');
+const { promsterPlugin } = require('@promster/fastify');
```

Comment:
Is this a naming convention in the community or could we also name this just `promster`? Maybe also just a fn so `createPromsterPlugin`. This would give us the ability to configure it which now we can't?

Reply:
it's not, so `promster` should be fine

we can configure it' as fastify passes options to plugin for us in this way:
```
fastify.register(promsterPlugin, {
    labels: ["version"],
    getLabelValues: function() {
      return { version: config.get("env.version") }
    },
    accuracies: ["ms"],
    metricTypes: [
      "httpRequestsTotal",
      "httpRequestsSummary",
      "httpRequestsHistogram",
    ],
  })
```